### ISSUE TITLE
[ADE Support] ADE - GitHub Integration - Slack Internal Channel Notifications (Trigger: New Intakes)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#102851]
> [!NOTE]
> **QUICK SUMMARY:** This ticket is the implementation of the integration of Slack with the Slack GitHub App in the [accessibility-de-internal channel](https://dsva.slack.com/archives/C08B367JLLT/p1739225413836779).

## Issue Description

To improve awareness of new intakes, in addition to VFS teams notifying us in the accessibility-help Slack channel, adding in the Slack GitHub integration for `ADE Intake` labels, will help to promote that expanded awareness. 

<img width="376" alt="GitHub Issue created by Sara-amanda linking to previous of issue 102848." src="https://github.com/user-attachments/assets/e831e52c-1d8f-4044-90fe-bab8961a5000" />

## About the Team

- Team name: ADE
- OCTO product owner: Martha Wilkes
- Product manager: Naomi Eke
- Slack channel: [accessibility-de-internal](https://dsva.slack.com/archives/C08B367JLLT)
- Dedicated accessibility specialist on your team (if you have one):
- Accessibility Champ (if you have one): `ALL`

### Supporting Artifacts 
- [Slack Channel for accessibility-de-internal](https://dsva.slack.com/archives/C08B367JLLT)
- [Notification Automation Message 5:10 p.m. EST Feb. 10, 2025](https://dsva.slack.com/archives/C08B367JLLT/p1739225413836779)
- [Test integration success 5:19 p.m. EST Feb. 10, 2025](https://dsva.slack.com/archives/C08B367JLLT/p1739225972537209)

```[tasklist]
### ADE Tasks
- [x] Add integration for GitHub to channel
- [x] Remove default triggers: Enabled by default and can be disabled:
- [x] **Keep**: `issues`
- [x] **Remove**: `pulls`, `commits`, `releases`, `deployments`
- [x] Trigger notifications for any new tickets with the label `ADE Intake`
- [x] Test implementation of Slack App for GitHub
- [ ] Notify Martha and team in channel of update. 

```
```[tasklist]
### Acceptance Criteria
- [x] Slack integration is tested and works properly.
- [x] Removals of extraneous default notifications are complete and no longer showing.
- [x] Martha and team are notified.
```
```[tasklist]
## Ticket Updates (CAIA Internal)
- [x] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [x] _Label with `Actual` (level of effort it took to complete this ticket)_
- [x] _Update date in_ `Closed Date`
```

```[tasklist]
### Related Tickets
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/102848
- [ ] #ticketnumbergoeshere1
- [ ] #ticketnumbergoeshere2
- [ ] #ticketnumbergoeshere3
```
